### DISCUSSION TITLE
Italy - Calabria

### DISCUSSION THREAD

**Discussion by Raffaelemzz** [electricitymaps/electricitymaps-contrib#4938] [General]
One of the Italian bidding zone is absent: Calabria.
I suggest you use Terna as a source for the Italian market.
https://www.terna.it/it/sistema-elettrico/transparency-report/actual-generation

**Comment by VIKTORVAV99** [comment]
We currently aggregate IT-SO (Italy South) and IT-CA (Italy Calabria) in the ENTSO-E parser so there is no data missing even if the zone is missing from the map. This was a active choice at the time as IT-CA had no historical data but it might be worth revisiting now. I'll check in with the team and see what they think.
